js/local-storage.js

```diff
@@ -0,0 +1,25 @@
+const nameInput = document.getElementById('name');
+const emailInput = document.getElementById('email');
+const messageInput = document.getElementById('message');
```

Comment:
instead of getting each input one by one how about we target the form directly like using document.formname

Reply:
@Yinkusadel Thanks for the review
Concerning making use of document.formName, I know it lessens the writing of code but I feel the way I did this is more understanding for me in every way

Reply:
I agree with @Yinkusadel 

What you have done is very okay and fine.

You can improve it by getting the form first, using the same style used for the calculator theme logic, where you get the form first with a `document.formName`, you can then get all the inputs inside the form using the form.querySelector method if you want.